src/test/java/org/takes/rq/multipart/RqMultipartBaseTest.java

```diff
@@ -137,20 +145,20 @@ public void satisfiesEqualsContract() throws IOException {
     }
 
     /**
-     * RqMultipart.Base can throw exception on no closing boundary found.
+     * RqMultipartBase can throw exception on no closing boundary found.
      * @throws IOException if some problem inside
      */
     @Test(expected = IOException.class)
     public void throwsExceptionOnNoClosingBoundaryFound() throws IOException {
-        new RqMultipart.Base(
+        new RqMultipartBase(
```

Comment:
@yegor256 are you fine to accept such public API change or we need to change the approach?